### PR TITLE
Add an important note to 2.1.0 release docs

### PR DESCRIPTION
**Pull request by MikeAlhayek**

### CODE REVIEW

**Comment by Piedone** [comment]
This means the change is actually breaking :|. But this can be fixed with at least a redirecting action in a controller under the original name, or perhaps something clever with route overrides.

**Comment by MikeAlhayek** [comment]
@Piedone agreed! Thanks for pointing that out. I added an auto redirect to prevent the breaking change.

File: src/docs/releases/2.1.0.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ We've introduced a new feature called **External Authentication**. This feature
 
 This feature is only available on-demand and cannot be manually enabled or disabled. It is automatically enabled when a feature requiring external authentication is activated.
 
+!!! warning
+    If you're using a custom `Login.cshtml` view or `Login` template, make sure to update the external login form action. The `ExternalLogin` action has been moved from the `Account` controller to the `ExternalAuthentications` controller as of this update.
```
Comment (Piedone) [inline src/docs/releases/2.1.0.md:30]:
```suggestion
    If you're using a custom `Login.cshtml` view or `Login` template, make sure to update the external login form action. The `ExternalLogin` action has been moved from the `Account` controller to the `ExternalAuthentications` controller as of this update. While old routes will continue to work, they'll be removed in a future release.
```

File: src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs [hunk 2]
```diff
@@ -256,6 +256,19 @@ public async Task<IActionResult> ChangePassword(ChangePasswordViewModel model, s
     public IActionResult ChangePasswordConfirmation()
         => View();
 
+    /// <summary>
+    /// This action is retained for backward compatibility.
+    /// Both this action and <see cref="ExternalAuthenticationsStartupFilter"/> are scheduled for removal in version 3.
+    /// </summary>
+    /// <returns></returns>
```
Comment (Piedone) [inline src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs:263]:
```suggestion
```

File: src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs [hunk 3]
```diff
@@ -256,6 +256,19 @@ public async Task<IActionResult> ChangePassword(ChangePasswordViewModel model, s
...
+    [HttpPost]
+    [AllowAnonymous]
+    [ValidateAntiForgeryToken]
+    public IActionResult ExternalLogin()
```
Comment (Piedone) [inline src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs:267]: What's the purpose of this action, exactly? In 2.0, it was `IActionResult ExternalLogin(string provider, string returnUrl = null)`, so I don't think routes to that one with parameters will match this one.

Comment (Piedone) [inline src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs:267]: Apart from the `ExternalLogin` action, there was also ``ExternalLogin` and `DefaultExternalLogin` that one could imagine referencing from custom code. Shouldn't those be taken care of too?

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs (hunk 3 above)
Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs:266]: I don't think `DefaultExternalLogin` is used by anyviews. This action is redirected to from the AccountController.Login which is still the case. So I don't think this is needed unless you think it is used in views that I am not aware of.

File: src/OrchardCore.Modules/OrchardCore.Users/Services/ExternalAuthenticationsStartupFilter.cs [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                    context.Request.RouteValues["area"] = "OrchardCore.Users";
+                }
+
+                await next();
+            });
+
+            next(builder);
+        };
+    }
+
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Users/Services/ExternalAuthenticationsStartupFilter.cs:45]:
```suggestion
```

File: src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs (hunk 3 above)
Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs:266]: Yeah, it actually matches despite the parameters not being there. Weird, but good then.

Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs:266]: Not just views, but any reference, including standard links. But looking into it, these don't seem to be affected.

**Review by Piedone — APPROVED** [review]

**Pull request merged**